WordPress/Classes/ViewRelated/Reader/ReaderRetryFailedImageView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ReaderRetryFailedImageView.swift
+//  WordPress
+//
+//  Created by Paul Von Schrottky on 10/5/19.
+//  Copyright © 2019 WordPress. All rights reserved.
+//
+
+import UIKit
+import Gridicons
+
+protocol ReaderRetryFailedImageDelegate: AnyObject {
+    func didTapRetry()
+}
+
+fileprivate class LinkTextView: UITextView {
+    override func selectionRects(for range: UITextRange) -> [UITextSelectionRect] {
+        return []
+    }
+
+    override func caretRect(for position: UITextPosition) -> CGRect {
+        return CGRect.zero.offsetBy(dx: .greatestFiniteMagnitude, dy: .greatestFiniteMagnitude)
+    }
+}
+
+class ReaderRetryFailedImageView: UIView {
+
+    @IBOutlet weak private var imageView: UIImageView! {
+        didSet {
+            let iconImage = Gridicon.iconOfType(.imageRemove)
+            imageView.image = iconImage.withRenderingMode(.alwaysTemplate)
+            imageView.tintColor = UIColor.textSubtle
+            imageView.contentMode = .scaleAspectFit
+        }
+    }
+
+    @IBOutlet weak private var textView: LinkTextView! {
+        didSet {
+            textView.delegate = self
+            textView.textDragInteraction?.isEnabled = false
+            textView.adjustsFontForContentSizeCategory = true
+        }
+    }
+
+    private let attributedString: NSAttributedString = {
```

Comment:
I sometimes come across an issue in GitHub where comments I write disappear when a PR is open in multiple tabs - so will recap here.
---
I went for a property because it seemed to succinctly encapsulate the creation of the attributed string. It's also a reference to an object that the `textView` also references, so memory shouldn't be an issue. Either way, I'll update to a function as I don't have a strong opinion on this.

Regarding the naming, since it's the only attributed string here, I felt the enclosing type name provided enough context.

Reply:
I missed the `static` part when reading this last night. There is nothing in the message specific to a particular instance so that could work here. I'll update this later today.

Reply:
Updated this to a `class` func because a `ReaderRetryFailedImageView` subclass (let's say for a embedded video) could want to override the text to change it to something different.